spec/spec.md

```diff
@@ -39,17 +39,27 @@ Presentation Exchange is a draft specification being developed within the Decent
 Term | Definition
 :--- | :---------
 Decentralized Identifier (DID) | Unique ID string and PKI metadata document format for describing the cryptographic keys and other fundamental PKI values linked to a unique, user-controlled, self-sovereign identifier in a target system (i.e. blockchain, distributed ledger).
-Subject | The entity that submits proofs to a Verifier to satisfy the requirements described in a Presentation Definition
-Verifier | The entity that defines what proofs they require from a Subject (via a Presentation Definition) in order to proceed with an interaction.
+Prover | The entity that submits proofs to a Verifier to satisfy the requirements described in a Presentation Definition
+Verifier | The entity that defines what proofs they require from a Prover (via a Presentation Definition) in order to proceed with an interaction.
 
 ## Presentation Definition
 
-Presentation Definitions are objects generate to articulate what proofs an entity requires to make a decision about an interaction with a Subject. Presentation Definitions are composed of inputs, which describe the forms and details of the proofs they require, and and optional set of selection rules, to allow Subjects flexibility in cases where many different types of proofs may satisfy an input requirement.
+Presentation Definitions articulate what proofs Verifier requires. Often, but not always, they help the Verifier decide how or whether to interact with a Prover. Presentation Definitions are composed of inputs, which describe the forms and details of the proofs they require, and and optional set of selection rules, to allow Subjects flexibility in cases where many different types of proofs may satisfy an input requirement.
 
 ::: example Presentation Definition - all features exercised
-```json
+```jsonc
 
 {
+    // TODO: The presentation definition MUST include a nonce. If it doesn't,
+    // anybody can replay the bytes they eventually receive, pretending to
+    // be the Prover. This is an absolute security requirement.
+
+    // TODO: how do we express boolean options? I can see how we do the trivial
+    // ones (pick any N out of M). But that's not what I'm asking about. When 
+    // I applied for citizenship for my daughter, I had to present any
+    // 2 proofs from category A, and any 1 proof from category B, OR I had
+    // to present 3 or more proofs from category B. I don't see a way to model
+    // that real-world use case. It think that's because we imagine the rules to only exist *within* a single requirement, never across requirements. And we don't imagine arbitrary combinations of booleans.
```

Comment:
I believe this TODO is addressed in PR #19

Reply:
as a general question, rather than TODOs in the spec, would it be better to raise these as issues?

Reply:
yes, its safe to merge a PR where the TODO links to an issue, but we should in general avoid merging PRs with TODOs that are not tracked in issues.

Reply:
Undoubtedly. However, I had half an hour to do a brain dump. I didn't want to open a dozen different issues; I wanted to ask a bunch of questions that provoked one or more conversations, and then bow back out again.